packages/base-dao/src/index.js

```diff
@@ -0,0 +1,5 @@
+const { createEntityDAO } = require('./baseDAO');
+
+module.exports = {
+  createEntityDAO
```

Comment:
It should also expose the runGenericDAOTests function right?

Reply:
true